src/packages/searchbar/searchbar.taro.tsx

```
@@ -131,61 +158,72 @@ export const SearchBar: FunctionComponent<
       </View>
     )
   }
   const clearaVal = (event: ITouchEvent) => {
-    if (disabled || readOnly) {
-      return
-    }
+    if (disabled || readOnly) return
     setValue('')
     forceFocus()
-    onChange && onChange?.('')
+    onChange && onChange('')
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)